calc/src/calc_fee.rs

```diff
@@ -220,3 +220,52 @@ fn new_u128(inner: &str) -> FixedU128 {
 fn new_legacy_128(inner: &str) -> Fixed128Legacy {
     Fixed128Legacy::from_parts(i128::from_str(inner).unwrap())
 }
+
+mod test_fees {
+    use super::Multiplier;
+    use super::Multiplier::{V2, V1, V0};
+    use super::new_u128;
+    use super::new_i128;
+    use super::new_legacy_128;
+
+    #[test]
+    fn multiplier_from_spec_name() {
+        let inner = "500000000";
```

Comment:
I'm no fan of calling this `inner`, what is it multiplier (coefficient)? 

fine, it's used widely in this crate....

Reply:
Yea I agree that it's not the most informative naming, and I'd be happy to update it to something more verbose in a follow up PR, but for this PR as you mentioned its probably wise to keep the continuity in the crate.